lib/datadog/tracing/contrib/redis/ext.rb

```diff
@@ -18,6 +18,10 @@ module Ext
           ### BRAZE MODIFICATION
           METRIC_RAW_COMMAND_LEN = 'redis.raw_command_length'.freeze
           METRIC_RESP_COMMAND_LEN = 'redis.raw_response_length'.freeze
+          METRIC_FILEPATH = 'filepath'.freeze
+          METRIC_CODEOWNER = 'codeowner'.freeze
```

Comment:
I wonder if these should be namespaced like what was done for the other Braze-added ones (i.e. `redis.filepath`)
@zachmccormick ?

Reply:
Is `redis` the right namespace, or would we want to use something like `platform.filepath`?

Reply:
this is for redis spans, so I would think `redis.filepath` or `redis.platform_filepath`
it probably only matters for aggregating/searching spans and not for looking at a single trace